gracefully handle invalid requests

We've observed `/info?txtAirPlay&txtRAOP` requests causing a stack trace with:
```
io.buoyant.router.RoutingFactory$UnknownDst: Unknown destination: invalid http request
```

It's preferable if we gracefully log more helpful info.

full stack trace for reference:
```
E 0511 17:38:26.529 THREAD60: Exception propagated to the root monitor!
io.buoyant.router.RoutingFactory$UnknownDst: Unknown destination: invalid http request
	at com.twitter.finagle.NoStacktrace(Unknown Source)
Caused by: java.lang.IllegalArgumentException: invalid http request
	at io.buoyant.router.http.DefaultIdentifier.apply(DefaultIdentifier.scala:39)
	at io.buoyant.router.http.DefaultIdentifier.apply(DefaultIdentifier.scala:15)
	at io.buoyant.router.RoutingFactory$RoutingService.apply(RoutingFactory.scala:87)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.tracing.ServerDestTracingProxy$$anon$1.apply(DestinationTracing.scala:32)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.service.DeadlineFilter.apply(DeadlineFilter.scala:176)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.filter.DtabStatsFilter.apply(DtabStatsFilter.scala:37)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.service.StatsFilter.apply(StatsFilter.scala:150)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.filter.ExceptionSourceFilter.apply(ExceptionSourceFilter.scala:42)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.filter.MkJvmFilter$$anon$1.apply(JvmFilter.scala:29)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.filter.ServerStatsFilter.apply(ServerStatsFilter.scala:43)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.tracing.AnnotatingTracingFilter.apply(TraceInitializerFilter.scala:118)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.http.filter.HttpNackFilter.apply(HttpNackFilter.scala:56)
	at com.twitter.finagle.http.filter.HttpNackFilter.apply(HttpNackFilter.scala:49)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(HttpTraceInitializer.scala:44)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(HttpTraceInitializer.scala:44)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter.sample(HttpTraceInitializer.scala:62)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter$$anonfun$apply$1$$anonfun$apply$2.apply(HttpTraceInitializer.scala:43)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter$$anonfun$apply$1$$anonfun$apply$2.apply(HttpTraceInitializer.scala:43)
	at com.twitter.util.Local.let(Local.scala:141)
	at com.twitter.finagle.context.Context$class.let(Context.scala:176)
	at com.twitter.finagle.context.MarshalledContext.let(Context.scala:232)
	at com.twitter.finagle.tracing.Trace$$anonfun$letTracerAndId$1.apply(Trace.scala:216)
	at com.twitter.util.Local.let(Local.scala:141)
	at com.twitter.finagle.context.Context$class.let(Context.scala:176)
	at com.twitter.finagle.context.LocalContext.let(LocalContext.scala:8)
	at com.twitter.finagle.tracing.Trace$.letTracerAndId(Trace.scala:215)
	at com.twitter.finagle.tracing.Trace$.letTracerAndNextId(Trace.scala:198)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter$$anonfun$apply$1.apply(HttpTraceInitializer.scala:42)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter$$anonfun$apply$1.apply(HttpTraceInitializer.scala:42)
	at com.twitter.finagle.tracing.Trace$.letIdOption(Trace.scala:178)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter.apply(HttpTraceInitializer.scala:41)
	at com.twitter.finagle.buoyant.linkerd.HttpTraceInitializer$server$Filter.apply(HttpTraceInitializer.scala:26)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.filter.MonitorFilter$$anonfun$apply$1.apply(MonitorFilter.scala:35)
	at com.twitter.finagle.filter.MonitorFilter$$anonfun$apply$1.apply(MonitorFilter.scala:35)
	at com.twitter.util.Future$$anonfun$monitored$1.apply$mcV$sp(Future.scala:156)
	at com.twitter.util.Monitor$$anonfun$apply$1.apply$mcV$sp(Monitor.scala:38)
	at com.twitter.util.Monitor$$anonfun$apply$1.apply(Monitor.scala:38)
	at com.twitter.util.Monitor$$anonfun$apply$1.apply(Monitor.scala:38)
	at com.twitter.util.Monitor$$anonfun$using$1.apply(Monitor.scala:110)
	at com.twitter.util.Monitor$.restoring(Monitor.scala:117)
	at com.twitter.util.Monitor$.using(Monitor.scala:108)
	at com.twitter.util.Monitor$class.apply(Monitor.scala:37)
	at com.twitter.util.Future$Monitored.apply(Future.scala:163)
	at com.twitter.util.Future$.monitored(Future.scala:155)
	at com.twitter.finagle.filter.MonitorFilter.apply(MonitorFilter.scala:34)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.filter.PayloadSizeFilter.apply(PayloadSizeFilter.scala:29)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at io.buoyant.linkerd.protocol.http.ErrorResponder$filter$.apply(ErrorResponder.scala:21)
	at io.buoyant.linkerd.protocol.http.ErrorResponder$filter$.apply(ErrorResponder.scala:15)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.http.filter.ServerContextFilter$$anonfun$apply$1.apply(ContextFilter.scala:17)
	at com.twitter.finagle.http.filter.ServerContextFilter$$anonfun$apply$1.apply(ContextFilter.scala:17)
	at com.twitter.finagle.http.codec.HttpContext$.read(HttpContext.scala:52)
	at com.twitter.finagle.http.filter.ServerContextFilter.apply(ContextFilter.scala:17)
	at com.twitter.finagle.http.filter.ServerContextFilter.apply(ContextFilter.scala:13)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.Service$$anon$1.apply(Service.scala:16)
	at com.twitter.finagle.http.filter.DtabFilter.apply(DtabFilter.scala:22)
	at com.twitter.finagle.http.filter.DtabFilter.apply(DtabFilter.scala:11)
	at com.twitter.finagle.Filter$$anon$1.apply(Filter.scala:79)
	at com.twitter.finagle.http.exp.HttpServerDispatcher.dispatch(HttpServerDispatcher.scala:58)
	at com.twitter.finagle.http.exp.HttpServerDispatcher.dispatch(HttpServerDispatcher.scala:19)
	at com.twitter.finagle.http.exp.GenSerialServerDispatcher$$anonfun$com$twitter$finagle$http$exp$GenSerialServerDispatcher$$loop$1$$anonfun$apply$1.apply$mcV$sp(ServerDispatcher.scala:55)
	at com.twitter.finagle.http.exp.GenSerialServerDispatcher$$anonfun$com$twitter$finagle$http$exp$GenSerialServerDispatcher$$loop$1$$anonfun$apply$1.apply(ServerDispatcher.scala:54)
	at com.twitter.finagle.http.exp.GenSerialServerDispatcher$$anonfun$com$twitter$finagle$http$exp$GenSerialServerDispatcher$$loop$1$$anonfun$apply$1.apply(ServerDispatcher.scala:54)
	at com.twitter.util.Local.let(Local.scala:141)
	at com.twitter.finagle.context.Context$class.let(Context.scala:176)
	at com.twitter.finagle.context.LocalContext.let(LocalContext.scala:8)
	at com.twitter.finagle.http.exp.GenSerialServerDispatcher$$anonfun$com$twitter$finagle$http$exp$GenSerialServerDispatcher$$loop$1.apply(ServerDispatcher.scala:53)
	at com.twitter.finagle.http.exp.GenSerialServerDispatcher$$anonfun$com$twitter$finagle$http$exp$GenSerialServerDispatcher$$loop$1.apply(ServerDispatcher.scala:48)
	at com.twitter.util.Future$$anonfun$flatMap$1.apply(Future.scala:986)
	at com.twitter.util.Future$$anonfun$flatMap$1.apply(Future.scala:985)
	at com.twitter.util.Promise$Transformer.liftedTree1$1(Promise.scala:112)
	at com.twitter.util.Promise$Transformer.k(Promise.scala:112)
	at com.twitter.util.Promise$Transformer.apply(Promise.scala:122)
	at com.twitter.util.Promise$Transformer.apply(Promise.scala:103)
	at com.twitter.util.Promise$$anon$1.run(Promise.scala:381)
	at com.twitter.concurrent.LocalScheduler$Activation.run(Scheduler.scala:178)
	at com.twitter.concurrent.LocalScheduler$Activation.submit(Scheduler.scala:136)
	at com.twitter.concurrent.LocalScheduler.submit(Scheduler.scala:207)
	at com.twitter.concurrent.Scheduler$.submit(Scheduler.scala:92)
	at com.twitter.util.Promise.runq(Promise.scala:350)
	at com.twitter.util.Promise.updateIfEmpty(Promise.scala:721)
	at com.twitter.util.Promise.update(Promise.scala:694)
	at com.twitter.util.Promise.setValue(Promise.scala:670)
	at com.twitter.concurrent.AsyncQueue.offer(AsyncQueue.scala:111)
	at com.twitter.finagle.netty3.transport.ChannelTransport.handleUpstream(ChannelTransport.scala:55)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.channel.SimpleChannelHandler.messageReceived(SimpleChannelHandler.java:142)
	at org.jboss.netty.channel.SimpleChannelHandler.handleUpstream(SimpleChannelHandler.java:88)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.channel.SimpleChannelHandler.messageReceived(SimpleChannelHandler.java:142)
	at com.twitter.finagle.netty3.channel.ChannelRequestStatsHandler.messageReceived(ChannelRequestStatsHandler.scala:32)
	at org.jboss.netty.channel.SimpleChannelHandler.handleUpstream(SimpleChannelHandler.java:88)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.handler.codec.http.HttpChunkAggregator.messageReceived(HttpChunkAggregator.java:145)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.messageReceived(SimpleChannelUpstreamHandler.java:124)
	at com.twitter.finagle.http.codec.RespondToExpectContinue.messageReceived(RespondToExpectContinue.scala:30)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.messageReceived(SimpleChannelUpstreamHandler.java:124)
	at com.twitter.finagle.http.codec.PayloadSizeHandler.messageReceived(PayloadSizeHandler.scala:24)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.handler.codec.http.HttpContentDecoder.messageReceived(HttpContentDecoder.java:108)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.handler.codec.http.HttpContentEncoder.messageReceived(HttpContentEncoder.java:82)
	at org.jboss.netty.channel.SimpleChannelHandler.handleUpstream(SimpleChannelHandler.java:88)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:296)
	at org.jboss.netty.handler.codec.frame.FrameDecoder.unfoldAndFireMessageReceived(FrameDecoder.java:459)
	at org.jboss.netty.handler.codec.replay.ReplayingDecoder.callDecode(ReplayingDecoder.java:536)
	at org.jboss.netty.handler.codec.replay.ReplayingDecoder.messageReceived(ReplayingDecoder.java:435)
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
	at org.jboss.netty.handler.codec.http.HttpServerCodec.handleUpstream(HttpServerCodec.java:56)
	at com.twitter.finagle.http.SafeHttpServerCodec.handleUpstream(Codec.scala:90)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
	at org.jboss.netty.channel.SimpleChannelHandler.messageReceived(SimpleChannelHandler.java:142)
	at com.twitter.finagle.netty3.channel.ChannelStatsHandler.messageReceived(ChannelStatsHandler.scala:68)
	at org.jboss.netty.channel.SimpleChannelHandler.handleUpstream(SimpleChannelHandler.java:88)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:559)
	at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:268)
	at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:255)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

--- a/linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/ErrorResponder.scala
+++ b/linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/ErrorResponder.scala
@@ -6,6 +6,7 @@ import com.twitter.finagle.http.{MediaType, Request, Response, Status}
 import com.twitter.logging.Logger
 import com.twitter.util.NonFatal
 import io.buoyant.router.RoutingFactory
+import java.net.URLEncoder
 
 object ErrorResponder extends Stack.Module0[ServiceFactory[Request, Response]] {
   val role = Stack.Role("ErrorResponder")
@@ -30,7 +31,7 @@ object ErrorResponder extends Stack.Module0[ServiceFactory[Request, Response]] {
         }
 
         val rsp = Response(status)
-        rsp.headerMap(Headers.Err) = e.getMessage
+        rsp.headerMap(Headers.Err) = URLEncoder.encode(e.getMessage, "ISO-8859-1")
         rsp.contentType = MediaType.Txt
         rsp.contentString = e.getMessage
         rsp

--- a/linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/ErrorResponderTest.scala
+++ b/linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/ErrorResponderTest.scala
@@ -1,0 +1,33 @@
+package io.buoyant.linkerd.protocol.http
+
+import com.twitter.finagle.{Service, ServiceFactory, Stack}
+import com.twitter.finagle.buoyant.linkerd.Headers
+import com.twitter.util.Future
+import com.twitter.finagle.http.{Request, Response, Status}
+import java.net.URLEncoder
+import io.buoyant.router.RoutingFactory
+import io.buoyant.test.Awaits
+import org.scalatest.FunSuite
+
+class ErrorResponderTest extends FunSuite with Awaits {
+
+  val svc = Service.mk[Request, Response] { _ =>
+    Future.exception(RoutingFactory.UnknownDst(Request(), new Exception(s"foo\nbar")))
+  }
+  val stk = ErrorResponder.toStack(
+    Stack.Leaf(ErrorResponder.role, ServiceFactory.const(svc))
+  )
+  val service = await(stk.make(Stack.Params.empty)())
+
+  test("returns BadRequest for UnknownDst exception") {
+    val rsp = await(service(Request()))
+    assert(rsp.status == Status.BadRequest)
+  }
+
+  test("correctly encodes error headers") {
+    val rsp = await(service(Request()))
+    val headerErr = rsp.headerMap(Headers.Err)
+    assert(!headerErr.contains("\n"))
+    assert(headerErr.contains(URLEncoder.encode("\n", "ISO-8859-1")))
+  }
+}

--- a/namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala
+++ b/namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala
@@ -82,7 +82,7 @@ object HttpControlService {
   }
 
   case class InvalidPathException(path: String, underlying: Exception)
-    extends Exception(s"Invalid path: $path\n${underlying.getMessage}", underlying)
+    extends Exception(s"Invalid path: $path / ${underlying.getMessage}", underlying)
 
   trait NsPathUri {
     val prefix: String

--- a/router/core/src/main/scala/io/buoyant/router/RoutingFactory.scala
+++ b/router/core/src/main/scala/io/buoyant/router/RoutingFactory.scala
@@ -10,7 +10,7 @@ object RoutingFactory {
   val description = "Performs per-request name binding"
 
   case class UnknownDst[Req](request: Req, cause: Throwable)
-    extends Exception(s"Unknown destination: ${cause.getMessage}", cause)
+    extends Exception(s"Unknown destination: $request / ${cause.getMessage}", cause)
     with NoStacktrace
 
   /**

--- a/router/core/src/test/scala/io/buoyant/router/RoutingFactoryTest.scala
+++ b/router/core/src/test/scala/io/buoyant/router/RoutingFactoryTest.scala
@@ -54,8 +54,14 @@ class RoutingFactoryTest extends FunSuite with Awaits with Exceptions {
     val tracer = new TestTracer()
     Trace.letTracer(tracer) {
       val service = mkService(pathMk = (_: Request) => Future.exception(Thrown))
-      assertThrows[RoutingFactory.UnknownDst[Request]] {
-        await(service(Request()))
+
+      val req = Request()
+      try {
+        await(service(req))
+        assert(false)
+      } catch {
+        case e: Exception => assert(e.getMessage.contains(req.toString))
+        case _: Throwable => assert(false)
       }
     }
   }

--- a/router/http/src/main/scala/io/buoyant/router/http/DefaultIdentifier.scala
+++ b/router/http/src/main/scala/io/buoyant/router/http/DefaultIdentifier.scala
@@ -33,7 +33,7 @@ case class DefaultIdentifier(
         case Some(host) if host.nonEmpty =>
           Future.value(mkPath(Path.Utf8("1.1", req.method.toString, host) ++ suffix(req)))
         case _ =>
-          Future.exception(new IllegalArgumentException("invalid http request"))
+          Future.exception(new IllegalArgumentException(s"${http.Version.Http11} request missing hostname: $req"))
       }
   }
 

--- a/router/http/src/main/scala/io/buoyant/router/http/TracingFilter.scala
+++ b/router/http/src/main/scala/io/buoyant/router/http/TracingFilter.scala
@@ -33,7 +33,7 @@ object TracingFilter extends SimpleFilter[Request, Response] {
     Trace.recordBinary("http.rsp.version", rsp.version.toString)
     rsp.contentLength.foreach(Trace.recordBinary("http.rsp.content-length", _))
     if (rsp.status.code >= 500 && rsp.status.code < 600) {
-      Trace.recordBinary("http.rsp.body", rsp.content.slice(0, 64 * 1000 /*64kb*/))
+      Trace.recordBinary("http.rsp.body", rsp.content.slice(0, 64 * 1000 /*64kb*/ ))
     }
     rsp.contentType.foreach(Trace.recordBinary("http.rsp.content-type", _))
     rsp.headerMap.get("transfer-encoding").foreach { te =>

